controllers/job.go

```diff
@@ -181,13 +185,34 @@ func (controller *JobController) Post(w http.ResponseWriter, r *http.Request, pa
 		return
 	}
 
-	updateData := struct{ NextState string }{}
+	updateData := struct {
+		NextState string
+		Timeout   time.Duration
+	}{}
 	err := json.NewDecoder(r.Body).Decode(&updateData)
 	if err != nil {
 		writeErr(w, err)
 		return
 	}
 
+	new_timeout := false
```

Comment:
```suggestion
	has_new_timeout := false
```
The `new_timeout` name feels like it should hold a timeout value, not a boolean.

Reply:
Good point. Updating it.